.github/ISSUE_TEMPLATE/new-feature-request.md

```diff
@@ -0,0 +1,29 @@
+---
+name: New Feature request
+about: Request a new feature for Skin
+title: "[NEW FEATURE]"
+labels: 'type: feature'
+assignees: ''
+
+---
+
+**Provide a desired name for the feature or module**
+A clear and concise name.
+
+**Describe the feature/module in detail**
+A description of how the feature/module is intended to work.
+
+**Screenshots**
+Add screenshots of the feature/module.
+
+**Figma Link**
+Add a link to the Figma page for the feature/module.
+
+**Prerequisites**
+
+- [ ] Successful design review with Design System Team.
+- [ ] Supports dark mode.
+- [ ] Supports responsive design.
+- [ ] Tokens have been provided (or already exist)
+- [ ] Successful accessibility review with a11y core team
```

Comment:
I would simplify the langue like, `passed accessibility review` or `Reviewed with a11y team`